### DISCUSSION TITLE
Github login for CoPilot crashes VS2022 17,8 ARM & previous version also

### DISCUSSION THREAD

**Discussion by rmilyo** [community/community#75631] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Got Copilot running VS Code, but VS2022 ARM 17,8  crashes & freezes when attempting to login w/ Github copilot account,... endless script errors -- previous VS2022 did same.
![Screenshot 2023-11-15 124614](https://github.com/community/community/assets/148013085/df58d04a-a9ff-415c-a9a8-83fe048ca762)

**Comment by ayim** [comment]
Hi @rmilyo, try setting this option to System web browser.
![image](https://github.com/community/community/assets/10951869/b3e07d30-a9be-494d-80af-0f5c1fdb984e)

**Reply by rmilyo** [reply]
Suggestion did not work; both installs already set to embedded web browser.  Just in case, I added Node.js and Python packages to VS2022 since I normally don't do web-based work, but no improve there, either.
Maybe something in how I had both VS installs configured beforehand, since both Intel and ARM installs have the same issue?

I’m going to wipe an old laptop & start with minimum VS2022 install to try to troubleshoot.

**Reply by ayim** [reply]
To confirm, when you set it to System Web Browser, is it STILL sending you to the browser that renders the page kind of in an Internet Explorer way above?

Expectation is setting to System Web Browser will use Microsoft Edge / Google Chrome / whatever you have set to default

**Comment by rmilyo** [comment]
FYI, I clicked "No" to dozens of script error dialogs and eventually managed to set up my Github account and validate CoPilot activation on my VS 2022 ARM install.  As to why I get all the script errors, I'll still try the approach I mentioned earlier, but it's got to be on the back burner now.

**Comment by rmilyo** [comment]
FYI - still have setting to external default browser, but last VS2022 and Github updates seem to have resolved my login and authentication issues with VS2022.  Will try internal browser settings & inform if it still is an issue.
Thank you.
